engine/flink/model-util/src/main/java/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/AggregateHelper.java

```diff
@@ -0,0 +1,64 @@
+package pl.touk.nussknacker.engine.flink.util.transformer.aggregate;
+
+import pl.touk.nussknacker.engine.api.ParamName;
+import pl.touk.nussknacker.engine.api.definition.DualParameterEditor;
+import pl.touk.nussknacker.engine.api.definition.FixedExpressionValue;
+import pl.touk.nussknacker.engine.api.definition.FixedValuesParameterEditor;
+import pl.touk.nussknacker.engine.api.definition.SimpleParameterEditor;
+import pl.touk.nussknacker.engine.api.editor.DualEditorMode;
+import scala.collection.JavaConverters;
+import scala.collection.JavaConverters$;
+
+import java.util.Arrays;
+import java.util.Map;
+
+// This class is in Java, because constants are used in expressions in editors - see
+// `pl.touk.nussknacker.engine.flink.util.transformer.aggregate.SlidingAggregateTransformerV2`. and scala objects are
+// not good for that. Be aware that. If you add some new aggregator please add it also there to make sure that it will
+// be available in selectbox.
+public class AggregateHelper {
+
+    public static final SimpleParameterEditor SIMPLE_EDITOR = new FixedValuesParameterEditor(JavaConverters.collectionAsScalaIterableConverter(Arrays.asList(
```

Comment:
What kind of performance yo have in mind? Because it is computed once... Enums would be ok, but then some more complex aggregates like `CardinalityAggregator` would be in the same place with basic once and I'm not a fun of this idea

Reply:
I forgot it's not LazyParameter :)